webv2/uniqueid/uniqueid_assignment.go

```diff
@@ -258,3 +258,50 @@ func InitObjectId() {
 	sessionState := session.GetSessionState()
 	sessionState.Counter.ObjectId = "0"
 }
+
+//CopyUniqueIdToSpannerTable copy ids from source table to spanner table content
+func CopyUniqueIdToSpannerTable(conv *internal.Conv, spannertablename string) {
+	sourcetablename := conv.ToSource[spannertablename].Name
+	spannertable := conv.SpSchema[spannertablename]
```

Comment:
technically conv.SpSchema[spannertablename] should be empty because we are restoring it

Reply:
This function is used only to assigned ID. Spanner table is already restore without before calling this function.